data/debug-cli/docker_debug.yaml

```diff
@@ -0,0 +1,318 @@
+command: docker debug
+short: Get a shell into any container or image. Replacement for docker exec.
+long: |-
+  > **Note**
+  >
+  > Docker Debug requires a [Pro, Team, or Business subcription](/subscription/details/).
+  > You must [sign in](/desktop/get-started/) to use this command.
+
+  Docker debug allows you to follow best practices and keep your images small and secure so they only contain the bare minimum to run your application.
+  That is it helps you to create and work with slim images/containers as they often are very hard to debug because all tools have been removed 
+  and thus standard debug approaches like `docker exec -it my-app bash` won't work.
+  Docker debug also helps you debug any other image or container.
+  With `docker debug` you can get a debug shell into any container/image, even if it doesn't contain a shell. 
+  You don't need to modify the image for debugging but also `docker debug` won't modify it either however it still allows you to debug it.
```

Comment:
This changes too much the original intention of the sentence. Does `You don't need to modify the image to use Docker Debug. 
  However, using Docker Debug still won't modify your image.` work?

The point here is that the user doesn't need to do anything to use Docker Debug. In the interviews many people wondered if that's because it does some modification automatically. So I'd like to emphasize that also using Docker Debug leaves the original image unchanged.

Reply:
Yep, that lgtm. My concern was "but" didn't work there and the sentence ran on.